build/build-sdk-images/csharp/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+#Is obsolete if artifacts are not needed when NuGet publish is written.
+#I'm still not sure how the artifacts are used, using `publish` for self-contained release packages.
```

Comment:
My intention was to include it for distributing binaries - C#'s bins still work cross-platform when built against .NET Standard. I assumed so when I found the build script existed for other languages as well.

The `publish` command in this fashion builds the code & packs it with it's dependencies in a single folder ready for deployment, as opposed to `build` which doesn't pack the dependencies.

In the end, it's not needed for testing nor for publishing on NuGet, if SDK builds aren't distributed then it's obsolete.
Shall I remove it?

Reply:
If the plan is to never distribute binaries, then yeah - probably makes sense to get rid of it.
@pooneh-m you know this space way better than I do -- do you have opinions here?

As an aside - you'll need to rebase against master to pass the e2e tests, since there were updates to the backing CRDs.

Reply:
That would be true. 
I see the C++ SDK bins being distributed for Linux, I don't know what would be needed to do the same for the C# SDK atm, so maybe I should remove it until it's planned or implemented? 
Could omit it all-together & go straight for NuGet as well.

I'll wait for confirmation on this one.

Will rebase 👌

Reply:
Yeah, I think let's remove this for now.